stores/dynamodb/src/entities/message.ts

```diff
@@ -0,0 +1,143 @@
+import { Entity } from 'electrodb';
+import { baseAttributes } from './utils';
+
+export const messageEntity = new Entity({
+  model: {
+    entity: 'message',
+    version: '1',
+    service: 'mastra',
+  },
+  attributes: {
+    entity: {
+      type: 'string',
+      required: true,
+    },
+    ...baseAttributes,
+    id: {
+      type: 'string',
+      required: true,
+    },
+    threadId: {
+      type: 'string',
+      required: true,
+    },
+    content: {
+      type: 'string',
+      required: true,
+      // Stringify content object on set if it's not already a string
+      set: (value?: string | void) => {
+        if (value && typeof value !== 'string') {
+          return JSON.stringify(value);
+        }
+        return value;
+      },
+      // Parse JSON string to object on get ONLY if it looks like JSON
+      get: (value?: string) => {
+        if (value && typeof value === 'string') {
+          try {
+            // Attempt to parse only if it might be JSON (e.g., starts with { or [)
+            if (value.startsWith('{') || value.startsWith('[')) {
+              return JSON.parse(value);
+            }
+          } catch (e) {
+            // Ignore parse error, return original string
+            return value;
+          }
+        }
+        return value;
+      },
+    },
```

Comment:
Another thing to note is that @TylerBarnes is making some changes to message formats, so this table might look different. But that might be something we can change at a later time.

Reply:
Actually, the message format should remain the same when inserting into the db, so this is good.